schema.graphql

```diff
@@ -11578,6 +11578,11 @@ type Discussion implements Closable & Comment & Deletable & Labelable & Lockable
   """
   viewerCanDelete: Boolean!
 
+  """
```

Comment:
This property does not appear in the [public schema](https://docs.github.com/en/graphql/overview/public-schema)